### PR TITLE
Update hardware information gathered by SUSEConnect

### PR DESCRIPTION
**Pull request by mssola**
As part of our last epic, we have introduced more parameters which are being gathered and sent to scc.suse.com. The public documentation should reflect these new parameters being sent.

### CODE REVIEW

**Comment by taroth21** [comment]
@mssola : Thanks for the PR (and sorry for not acting earlier, this probably got overlooked during the vacation period). @tbazant : Could you please take a look and review?

File: suseconnect-visibility/xml/art_suseconnect_visibility.xml [hunk 1]
```diff
@@ -289,6 +289,41 @@
...
+          </row>
+          <row>
+            <entry>uname</entry>
+            <entry>string</entry>
+            <entry>Linux lair 6.9.7-1-default #1 SMP PREEMPT_DYNAMIC Fri Jun 28 05:50:47 UTC 2024 (a5efffa) x86_64 x86_64 x86_64 GNU/Linux</entry>
+          </row>
+          <row>
+            <entry>Architecture specifics</entry>
+            <entry>map</entry>
+            <entry>Depends on the architecture. It includes parameters such as devicetree information, virtualization type on PowerPC or System Z, among others</entry>
```
Comment (tbazant) [inline suseconnect-visibility/xml/art_suseconnect_visibility.xml:305]:
```suggestion
            <entry>Depends on the architecture. It includes parameters such as device-tree information or virtualization type on PowerPC or System Z.</entry>
```

File: suseconnect-visibility/xml/art_suseconnect_visibility.xml [hunk 2]
```diff
@@ -289,6 +289,41 @@
...
+          </row>
+          <row>
+            <entry>SAP</entry>
+            <entry>list</entry>
+            <entry>
+             <para>
+              For each SAP installation: system_id (string) and instance_types (list of strings)
```
Comment (tbazant) [inline suseconnect-visibility/xml/art_suseconnect_visibility.xml:312]:
```suggestion
              <literal>system_id</literal> (string) and <literal>instance_types</literal> (list of strings) for each SAP installation.
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Just minor suggestions

**Pull request merged**